### PR TITLE
Bugfix/api-docs

### PR DESCRIPTION
**Pull request by jeesunikim**
* [AttributeTable's weird border-line](https://app.asana.com/0/1119309091112078/1170049853594169)
* [H1's odd margin-top & padding-top & Headers font size does not match the design](https://app.asana.com/0/1119309091112078/1170049853594171)
* Updating `SingleApiReference`. I assume it's just a copycat of `ApiReference.js`, @vcarl?

### CODE REVIEW

**Comment by vcarl** [comment]
@jeesunikim `SingleApiReference` needs to only render a single page from API reference, it's the SEO scheme so we can associate the content with a URL.

**Comment by jeesunikim** [comment]
>

**Comment by jeesunikim** [comment]
Updated @vcarl

File: src/templates/ApiReference.js [hunk 1]
```diff
@@ -267,25 +276,27 @@ const ReferenceSection = React.memo(
...
-
-          <NestedRow>
-            <MDXRenderer>{body}</MDXRenderer>
-          </NestedRow>
-          <HorizontalRule />
-        </section>
-      </Route>
+      <SectionEl>
+        <Route originalFilePath={relativePath} path={path}>
+          <section>
```
[the remark this answers is not in the document]
Comment (jeesunikim) [reply src/templates/ApiReference.js]: Updated - Had to add `React.Fragment` since `<Route/>` was expecting only one child

**Pull request merged**